header.php

```diff
@@ -14,7 +14,7 @@
 <html <?php language_attributes(); ?>>
 <head>
 	<meta charset="<?php bloginfo( 'charset' ); ?>" />
-	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
```

Comment:
I've opened a separate PR just for this one change: https://github.com/WordPress/twentynineteen/pull/199